src/main/java/org/ow2/proactive/connector/iaas/cloud/provider/vmware/VMWareProvider.java

```diff
@@ -470,7 +463,7 @@ private VirtualMachineRelocateSpec inferRelocateSpecsFromImageArgument(String im
             try {
                 return vmToClone.getResourcePool();
             } catch (RemoteException e) {
-                throw new RuntimeException("Unable to retrieve destination resource pool for VM:");
+                throw new RuntimeException("Unable to retrieve destination resource pool for VM:", e);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define and throw a dedicated exception instead of using a generic one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS00112)